cocos/2d/CCScene.cpp

```diff
@@ -209,6 +209,8 @@ void Scene::render(Renderer* renderer, const Mat4& eyeTransform)
         // And it is important that the change is "permament", because the matrix might be used for calculate
         // culling and other stuff.
         camera->setAdditionalTransform(eyeTransform.getInversed());
+        if (eyeProjection)
+            camera->setAdditionalProjection(*eyeProjection * camera->getProjectionMatrix().getInversed());
```

Comment:
probably we don't need to pass the `eyeProjection` then.
If you multiply `eyeTransform` with `eyeProjection` before calling `Scene::render()`, it should work, right?